<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResultWidget</class>
 <widget class="QWidget" name="SearchResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_location">
       <property name="text">
        <string>Results:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveSearchResults">
       <property name="maximumSize">
        <size>
         <width>122</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Save to file...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_display_all">
       <property name="text">
        <string>display all on map</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Distance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Coordinates</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Buys</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sells</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
